src/login/travel-document-onboarding/TravelDocument.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {View} from 'react-native';
+import {TCardAsTravelDocument} from '@atb/login/travel-document-onboarding/TCardAsTravelDocument';
+import {PhoneAsTravelDocument} from '@atb/login/travel-document-onboarding/PhoneAsTravelDocument';
+import {StyleSheet} from '@atb/theme';
+import {StaticColorByType} from '@atb/theme/colors';
+import {useMobileTokenContextState} from '@atb/mobile-token/MobileTokenContext';
+import {NoTravelDocument} from '@atb/login/travel-document-onboarding/NoTravelDocument';
+
+const themeColor: StaticColorByType<'background'> = 'background_accent_0';
+
+const TravelDocument = () => {
```

Comment:
Objectively I also think TravelDocument is a better sounding name. However MobileToken is a well established terminology in AtB now, so I would just keep using that to avoid confusion.

Reply:
updated 👍